app/views/layout.erb

```diff
@@ -2,14 +2,29 @@
 <html>
   <head>
     <title>Fwitter</title>
+    <link href="/css/main.css" rel="stylesheet">
   </head>
   <body>
+    <nav>
+      <a href="/">Home</a>
+      <% if session[:user_id].present? %>
```

Comment:
You can use your `logged_in?` helper method here, which is preferable to referring to the session object directly.

Reply:
Yea not going to lie while styling.. I coded wam bam as it wasn't a focus of the lab and just wanted to have a little fun but I'm sure I wouldv'e if I was really throwing all focus on it see below from my sinatra project where I'm using it exactly like that :)

https://github.com/AlphaDaniel/snalgos/blob/a7bdc78edceb11f62a7dff6249fd49c8617e4083/app/views/layout.erb#L37